yt_dlp/extractor/funker530.py

```diff
@@ -0,0 +1,75 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import re
+
```

Comment:
`unicode_literals` isn't necessary

and IMO we should use the (non-fatal) `_search_regex` method instead of importing `re`